web/yo/app/scripts/services/mainutils.js

```diff
@@ -285,18 +285,29 @@ angular.module('oncokbApp')
          * @return {*|h.promise|promise|r.promise|d.promise} Promise
         * */
         function sendEmail(sendTo, subject, content) {
-            var deferred = $q.defer();
-            var param = {sendTo: sendTo, subject: subject, content: content};
-            DatabaseConnector.sendEmail(
-                param,
-                function(result) {
-                    deferred.resolve(result);
-                },
-                function(result) {
-                    deferred.reject(result);
-                }
-            );
-            return deferred.promise;
+            if(content){
```

Comment:
Also do a sendTo check.

Reply:
I think check might not be needed here, because there is a promise. If sendTo is null, the return should be reject. If I did check here, there might not be reminder.

Reply:
You should handle the empty/undefined case and reject the promise manually. The promise you referenced is the one actually sending to backend for the service. But the whole purpose of checking is to avoid that.